dbt/adapters/athena/connections.py

```diff
@@ -106,6 +108,8 @@ def inner():
                 s3_staging_dir=s3_staging_dir,
                 cache_size=cache_size,
                 cache_expiration_time=cache_expiration_time,
+                result_reuse_enable=result_reuse_enable,
```

Comment:
as https://github.com/dbt-athena/dbt-athena/pull/185/files#r1145038838

Reply:
thx, removed. Should be fixed later
```
Signature of method 'AthenaCursor.execute()' does not match signature of the base method in class 'Cursor' 
```